iib/workers/tasks/build_merge_index_image.py

```diff
@@ -196,7 +197,8 @@ def handle_merge_request(
     target_index_resolved = prebuild_info['target_index_resolved']
 
     with tempfile.TemporaryDirectory(prefix='iib-') as temp_dir:
-        if is_image_fbc(source_from_index_resolved):
+        is_fbc = is_image_fbc(source_from_index_resolved)
+        if is_fbc:
```

Comment:
Could we remove this in favor line 254?

Reply:
No because merge index image is not yet supported. It should be removed in https://github.com/release-engineering/iib/pull/343

Reply:
ok, I will check the next PR